docs/deployment.md

```diff
@@ -0,0 +1,105 @@
+# Deploying and updating the IDR VAE
+
+All deployment on the IDR VAE is done from the `master` branch of this repository using GitLab CI/CD, configured in [`.gitlab-ci.yml`](../.gitlab-ci.yml)
+
+
+## Helm charts
+
+Everything is deployed using Helm charts, managed with the Helmfile tool which is configured in [`helmfile.yaml`](../helmfile.yaml).
+
+Most charts are released upstream:
+  - JupyterHub
+  - Grafana
+  - Prometheus
+
+A few are custom written for this project.
+These are in the [`charts`](../charts/) directory:
+  - [`charts/idr-analysis-website`](../charts/idr-analysis-website/) is the static website content
+  - [`charts/jupyterhub-shared-scratch`](../charts/jupyterhub-shared-scratch) creates the shared scratch space for the VAE (user scratch spaces are automatically created by JupyterHub)
+
+
+## Configuration
+
+There are four JupyterHub deployments.
+All use the common configuration in [`jupyterhub-config.yaml`](../jupyterhub-config.yaml) with additional configuration or overrides in separate files.
+
+### 1. JupyterHub `staging`
+This is always deployed as soon as the `master` branch is updated, and is fully configured by [`jupyterhub-config.yaml`](../jupyterhub-config.yaml).
```

Comment:
Could see this being renamed to `-staging.yaml`

Reply:
This is part of the standard config used by all jupyterhub deployments, the rest add to it.